worker/codalabworker/download_util.py

```diff
@@ -26,7 +26,7 @@ def get_target_info(bundle_path, uuid, path, depth):
     """
     final_path = _get_normalized_target_path(bundle_path, uuid, path)
 
-    if not os.path.islink(final_path) and not os.path.exists(final_path):
+    if not os.path.exists(final_path):
```

Comment:
Was there a reason this was changed?

Reply:
`exists` already returns false if `islink` returns false, so this is avoiding unnecessary work.